codalab/lib/worksheet_util.py

```diff
@@ -617,11 +618,11 @@ def raise_genpath_usage_error():
 
             for item_index, bundle_info in bundle_infos:
                 if is_missing(bundle_info):
-                    interpreted_items.append({
-                        'mode': TYPE_MARKUP,
-                        'interpreted': 'ERROR: cannot access bundle',
-                        'properties': {},
-                    })
+                    interpreted_items.append(
```

Comment:
Should call these `blocks` instead of `interpreted_items` to avoid confusion...

Reply:
sounds good!

Reply:
Still need to do this, right?

Reply:
yeah, wanted to simplify the diff view by doing this last